src/helpers/notification.helper.js

```diff
@@ -0,0 +1,105 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const {
+  constants: {
+    TOKEN_AUTHORITY_MASK,
+    TOKEN_MINT_MASK,
+    TOKEN_MELT_MASK,
+  },
+  dateFormatter,
+  wallet: oldWallet,
+} = require('@hathor/wallet-lib');
+
+/**
+ * @typedef {Object} Authorities
+ * @property {number} mint
+ * @property {number} melt
+ */
+
+/**
+ * @typedef TokenBalance
+ * @property {{locked: number, unlocked: number}} tokens
+ * @property {{locked: Authorities, unlocked: Authorities}} authorities
+ */
+
+/**
+ * Calculate the wallet balance for a tx
+ *
+ * @param {HathorWallet} wallet - The wallet we wish to calculate balance for.
+ * @param {Object} tx
+ *
+ * @returns {TokenBalance}
+ */
+function getWalletBalanceForTx(wallet, tx) {
```

Comment:
I'm starting to think if this method should be in the lib. It seems a bit more complete than the one we have in the wallet class `getTxBalance`.